src/modules/SX126x/SX126x.cpp

```diff
@@ -288,6 +290,14 @@ int16_t SX126x::receive(uint8_t* data, size_t len) {
     float symbolLength = (float)(uint32_t(1) << _sf) / (float)_bwKhz;
     timeout = (uint32_t)(symbolLength * 100.0 * 1000.0);
 
+      // set implicit mode and expected len if applicable
+      if (_headerType == SX126X_LORA_HEADER_IMPLICIT) {
+          state = setPacketParams(_preambleLength, _crcType, _implicitLen, _headerType);
+          if(state != ERR_NONE) {
+            return(state);
+          }
+      }
+
```

Comment:
This is not necessary, there's a call to `startReceive` later, which in turn will call `startReceiveCommon` that already calls setPacketParams

Reply:
Quite right, missed that -- thanks.